java/bulk-import/bulk-import-starter/src/main/java/sleeper/bulkimport/starter/executor/EmrServerlessPlatformExecutor.java

```diff
@@ -0,0 +1,112 @@
+ * limitations under the License.
+ */
+package sleeper.bulkimport.starter.executor;
+
+import software.amazon.awssdk.services.emrserverless.EmrServerlessClient;
+import software.amazon.awssdk.services.emrserverless.model.ConfigurationOverrides;
+import software.amazon.awssdk.services.emrserverless.model.JobDriver;
+import software.amazon.awssdk.services.emrserverless.model.MonitoringConfiguration;
+import software.amazon.awssdk.services.emrserverless.model.S3MonitoringConfiguration;
+import software.amazon.awssdk.services.emrserverless.model.SparkSubmit;
+import software.amazon.awssdk.services.emrserverless.model.StartJobRunRequest;
+
+import sleeper.bulkimport.job.BulkImportJob;
+import sleeper.configuration.properties.instance.InstanceProperties;
+
+import static sleeper.configuration.properties.instance.CommonProperty.ID;
+import static sleeper.configuration.properties.instance.EMRServerlessProperty.BULK_IMPORT_EMR_SERVERLESS_CLASS_NAME;
+import static sleeper.configuration.properties.instance.EMRServerlessProperty.BULK_IMPORT_EMR_SERVERLESS_DRIVER_CORES;
+import static sleeper.configuration.properties.instance.EMRServerlessProperty.BULK_IMPORT_EMR_SERVERLESS_DRIVER_MEMORY;
+import static sleeper.configuration.properties.instance.EMRServerlessProperty.BULK_IMPORT_EMR_SERVERLESS_EXECUTOR_CORES;
+import static sleeper.configuration.properties.instance.EMRServerlessProperty.BULK_IMPORT_EMR_SERVERLESS_EXECUTOR_INSTANCES;
+import static sleeper.configuration.properties.instance.EMRServerlessProperty.BULK_IMPORT_EMR_SERVERLESS_EXECUTOR_MEMORY;
+import static sleeper.configuration.properties.instance.EMRServerlessProperty.BULK_IMPORT_EMR_SERVERLESS_JAVA_HOME;
+import static sleeper.configuration.properties.instance.SystemDefinedInstanceProperty.BULK_IMPORT_BUCKET;
+import static sleeper.configuration.properties.instance.SystemDefinedInstanceProperty.BULK_IMPORT_EMR_SERVERLESS_APPLICATION_ID;
+import static sleeper.configuration.properties.instance.SystemDefinedInstanceProperty.BULK_IMPORT_EMR_SERVERLESS_CLUSTER_NAME;
+import static sleeper.configuration.properties.instance.SystemDefinedInstanceProperty.BULK_IMPORT_EMR_SERVERLESS_CLUSTER_ROLE_ARN;
+import static sleeper.configuration.properties.instance.SystemDefinedInstanceProperty.CONFIG_BUCKET;
+
+/**
+ * A {@link PlatformExecutor} which runs a bulk import job on EMR Serverless.
+ */
+public class EmrServerlessPlatformExecutor implements PlatformExecutor {
+    private final EmrServerlessClient emrClient;
+    private final InstanceProperties instanceProperties;
+
+    public EmrServerlessPlatformExecutor(EmrServerlessClient emrClient,
+                                         InstanceProperties instanceProperties) {
+        this.emrClient = emrClient;
+        this.instanceProperties = instanceProperties;
+    }
+
+    @Override
+    public void runJobOnPlatform(BulkImportArguments arguments) {
+        String bulkImportBucket = instanceProperties.get(BULK_IMPORT_BUCKET); // Todo change to own
```

Comment:
I'm not sure this todo is needed - we use that bucket in all bulk import stacks

Reply:
Removed